addon/templates/components/frost-modal-binding.hbs

```diff
@@ -4,8 +4,9 @@
     send=(hash
       animation=animation
       body=(component modal
-        classModifier=classModifier
+        class=class
         params=params
+        classNames=classNames
```

Comment:
What is your intended outcome of this usage?

Reply:
To get `frost-modal-form` class onto the `to-elsewhere` component, rather than `frost-modal-form`